### PR TITLE
bandwidth: support firefox

### PR DESCRIPTION
**Pull request by fippo**
adds Firefox support for the bandwidth sample.

@nils-ohlmeier re https://bugzilla.mozilla.org/show_bug.cgi?id=1359854 which i could reproduce with this 
-- bonus points if you can also review this ;-)

### CODE REVIEW

**Comment by fippo** [comment]
odd, it seems to take half the bandwidth i expect...

File: src/content/peerconnection/bandwidth/js/main.js
Comment (nils-ohlmeier) [inline src/content/peerconnection/bandwidth/js/main.js]: Wrong indentation.

File: src/content/peerconnection/bandwidth/js/main.js
Comment (nils-ohlmeier) [inline src/content/peerconnection/bandwidth/js/main.js]: How about we make this future proof and copy over the IP4 or IP6 as well, rather then assuming the Internet will keep on running on the deprecated IP version for ever.

File: src/content/peerconnection/bandwidth/js/main.js
Comment (nils-ohlmeier) [inline src/content/peerconnection/bandwidth/js/main.js]: I think you want here now:
`sdp.replace(/b=TIAS:.*\r\n/,'').replace(/b=AS.*\r\n/,'')`

File: src/content/peerconnection/bandwidth/js/main.js
Comment (nils-ohlmeier) [inline src/content/peerconnection/bandwidth/js/main.js:117]: I'm pretty sure I have some code laying around which allows to use the bandwidth drop down for the initial OA. I'll merge and submit that once this is merged.

**Review by nils-ohlmeier — COMMENTED** [review]
Looks good with some small changes.

**Review by nils-ohlmeier — APPROVED** [review]
LGTM.
Appears to work with Firefox according to my testing.

File: src/content/peerconnection/bandwidth/js/main.js
Comment (KaptenJansson) [reply src/content/peerconnection/bandwidth/js/main.js:117]: SGTM

**Review by KaptenJansson — APPROVED** [review]
LGTM! Thanks for doing this, maybe it's worth sending a small PSA about this?

**Comment by fippo** [comment]
Nils already had a PSA for the bug

**Pull request merged**
